repository-data/application/src/main/resources/hcm-config/namespaces/hee/blockLinksReference.yaml

```diff
@@ -0,0 +1,92 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/blockLinksReference:
```

Comment:
The ticket `HEE-221` doesn't seem to describe about `title` and `description` overrides. Suggest you to check with @MattBrooksManifesto to see if these are required (in case if Editors need the ability to override them).

Reply:
Yes, have removed - can add back in if required in future. Commited on 548764f